packages/libs/eda/src/lib/map/analysis/hooks/standaloneMapMarkers.tsx

```diff
@@ -339,7 +347,9 @@ export function useStandaloneMapMarkers(
 
         const count =
           vocabulary != null // if there's an overlay (all expected use cases)
-            ? overlayValues.reduce((sum, { count }) => (sum = sum + count), 0)
+            ? overlayValues
+                .filter(({ binLabel }) => vocabulary.includes(binLabel))
```

Comment:
Here's the filter I suggested in Slack that makes sure the counts are correct for low-cardinality vars.